code/modules/mob/living/carbon/xenomorph/mutators/strains/spitter/marker.dm

```diff
@@ -0,0 +1,39 @@
+/datum/xeno_mutator/marker
+	name = "STRAIN: Spitter - Marker"
+	description = "You weaken your corrosive acid for pheremones to support your sisters and acid to weaken your foes"
+	flavor_description = "Help your sisters fight better"
+	cost = MUTATOR_COST_EXPENSIVE
+	individual_only = TRUE
+	caste_whitelist = list(XENO_CASTE_SPITTER)
+	mutator_actions_to_remove = list(
+		/datum/action/xeno_action/activable/xeno_spit,
+		/datum/action/xeno_action/onclick/charge_spit,
+		/datum/action/xeno_action/activable/spray_acid/spitter,
+	)
+	mutator_actions_to_add = list(
+		/datum/action/xeno_action/activable/xeno_spit/,
```

Comment:
Why does this remove and then re-add xeno_spit? Unnecessary.

Additionally there's a trailing /, but may as well just delete this line 14 and 9.